### PR TITLE
Create basic obstacle avoidance sketch

### LINKED ISSUE #9: Obstacle Avoidance
## User Story
As an owner of the car, I want the car to stop and fully turn around before hitting an obstacle so that I do not break the car.

## Acceptance Criteria: 
The car stops 5 cm before hitting an obstacle, it will also do a 180 degree turn before continuing to dance.

### PR DESCRIPTION
**Pull request by FayonaC**
Created basic sketch to make the SmartCar stop moving forwards before it collides with an obstacle.

Code is taken from Dimitrios Platis' platform workshop 2020-04-01.

Resolves: #9

### CODE REVIEW

File: src/car/avoid/avoid.ino [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+SR04 sensor(TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);
+
+
+void setup() {
+
+}
+
+void loop(){
+    unsigned int distance = sensor.getDistance();
+    if (distance && distance < 20){
```
Comment (platisd) [inline src/car/avoid/avoid.ino]: It is considered a good practice unless a variable is a boolean, to be explicit about it. Namely in this case `distance != 0`

**Review by platisd — APPROVED** [review]
Minor (style) suggestion, otherwise it looks correct.
Just keep in mind that once there is no more obstacle in the way, the car will start moving again.

**Review by JenniNord — CHANGES_REQUESTED** [review]
Make the suggested change before approval.

**Review by Victori4 — APPROVED** [review]
Good style improvement distance in if statement

**Review by JenniNord — APPROVED** [review]

**Pull request merged**